osx-utils/performous-app-build.sh

```diff
@@ -3,6 +3,13 @@ set -o errexit
 # the very first step is to check that dylibbundler exists,
 # without it the bundle would be broken
 
+PREFIXDIR="/opt/local" # By default, the default prefix for macports, change this if you're using a different path or package manager.
+DEPLOYMENT_TARGET="10.9" # Change this if you want to target a different version of macOS.
+MAKE_JOBS=8 # Change this to change the number of jobs spawned.
```

Comment:
You can change this into `MAKE_JOBS=$(cat /proc/cpuinfo | awk '/^processor/{print $3}' | wc -l)` which will count the logical cpu cores given by the proc system which is most likely the result you want to have.

Reply:
Besides this it looks good to me